src/metabase/util/compress.clj

```diff
@@ -0,0 +1,54 @@
+(ns metabase.util.compress
```

Comment:
Why's this needed for defendpoint2? is it gzipping somehow? I would've thought that would be handled by a lower layer of the network stack